roles/adoption_osp_deploy/tasks/login_registries.yml

```diff
@@ -0,0 +1,50 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Activate red hat subscription
+  when:
+    - cifmw_adoption_osp_deploy_rhsm_org is defined
+    - cifmw_adoption_osp_deploy_rhsm_key is defined
+  become: true
+  no_log: true
+  ansible.builtin.command: >-
+    subscription-manager register --force
+    --org "{{ cifmw_adoption_osp_deploy_rhsm_org }}"
```

Comment:
how will this work though i mean the secret cannot live in this repo as it is not trusted/config repo.

Reply:
this is not intended to be used in CI jobs, this is here in case someone wants to run the deployment manually like I've been doing to test it. Once I start building the zuul jobs, I'll need to do something similar from a config repo